src/pages/public_registration/PublicRegistrationContent.tsx

```diff
@@ -86,6 +86,13 @@ export const PublicRegistrationContent = ({ registration }: PublicRegistrationCo
 
         <PublicGeneralContent registration={registration} />
 
+        {entityDescription?.alternativeTitles.und && (
+          <Box sx={{ borderTop: '1px solid', py: '1rem' }}>
+            <Typography variant="h3">{t('registration.public_page.alternative_title')}</Typography>
+            <Typography>{entityDescription?.alternativeTitles.und}</Typography>
```

Comment:
trenger ikke denne siden det allerede er sjekket på at det er en verdi her på linje 89
```suggestion
            <Typography>{entityDescription.alternativeTitles.und}</Typography>
```